src/mrpro/data/traj_calculators/KTrajectoryPulseq.py

```diff
@@ -54,13 +54,17 @@ def __call__(self, kheader: KHeader) -> KTrajectoryRawShape:
             raise ValueError('We currently only support constant number of samples')
         n_k0 = int(n_samples.item())
 
-        def reshape_pulseq_traj(k_traj: torch.Tensor, encoding_size: int):
-            k_traj *= encoding_size / (2 * torch.max(torch.abs(k_traj)))
+        def rescale_and_reshape_traj(k_traj: torch.Tensor, encoding_size: int):
+            if encoding_size > 1 and torch.max(torch.abs(k_traj)) > 0:
+                scaling_factor = encoding_size / (2 * torch.max(torch.abs(k_traj)))
+            else:
+                scaling_factor = 0
```

Comment:
Maybe add a comment here which describes that setting the value to 0 here is due to potential rounding errors and we don't have enough information to do better.